commons/src/main/java/org/infinispan/commons/marshall/ProtoStreamTypeIds.java

```diff
@@ -37,52 +40,52 @@
 
    // 2 byte Ids 128..16383
    // Commons range 128 -> 999
-   int COMMONS_START = 128;
-   int MEDIA_TYPE = COMMONS_START;
+   int COMMONS_LOWER_BOUND = 128;
+   int MEDIA_TYPE = COMMONS_LOWER_BOUND;
 
    // Core range 1000 -> 3999
-   int CORE_START = COMMONS_START + 872;
-   int EVENT_LOG_CATEGORY = CORE_START + 1;
-   int EVENT_LOG_LEVEL = CORE_START + 2;
-   int MARSHALLED_VALUE_IMPL = CORE_START + 3;
-   int META_PARAMS_INTERNAL_METADATA = CORE_START + 4;
-   int REMOTE_METADATA = CORE_START + 5;
+   int CORE_LOWER_BOUND = COMMONS_LOWER_BOUND + 872;
+   int EVENT_LOG_CATEGORY = CORE_LOWER_BOUND;
+   int EVENT_LOG_LEVEL = CORE_LOWER_BOUND + 1;
+   int MARSHALLED_VALUE_IMPL = CORE_LOWER_BOUND + 2;
+   int META_PARAMS_INTERNAL_METADATA = CORE_LOWER_BOUND + 3;
+   int REMOTE_METADATA = CORE_LOWER_BOUND + 4;
 
    // Counter range 4000 -> 4199
-   int COUNTERS_START = CORE_START + 3000;
-   int COUNTER_STATE = COUNTERS_START + 1;
+   int COUNTERS_LOWER_BOUND = CORE_LOWER_BOUND + 3000;
+   int COUNTER_STATE = COUNTERS_LOWER_BOUND;
 
    // Query range 4200 -> 4399
-   int QUERY_START = COUNTERS_START + 200;
-   int KNOWN_CLASS_KEY = QUERY_START + 1;
+   int QUERY_LOWER_BOUND = COUNTERS_LOWER_BOUND + 200;
+   int KNOWN_CLASS_KEY = QUERY_LOWER_BOUND;
 
    // Remote Query range 4400 -> 4599
-   int REMOTE_QUERY_START = QUERY_START + 200;
-   int PROTOBUF_VALUE_WRAPPER = REMOTE_QUERY_START + 1;
+   int REMOTE_QUERY_LOWER_BOUND = QUERY_LOWER_BOUND + 200;
+   int PROTOBUF_VALUE_WRAPPER = REMOTE_QUERY_LOWER_BOUND;
```

Comment:
We may want to move this into the 127 range as well. I believe when querying is enabled we replace WrappedByteArray with this instance instead.